com.unity.netcode.gameobjects/Components/NetworkAnimator.cs

```diff
@@ -5,12 +5,10 @@
 namespace Unity.Netcode.Components
 {
     /// <summary>
-    /// A prototype component for syncing animations
+    /// A prototype base-class component for syncing the state of Mecanim Animators
     /// </summary>
-    [AddComponentMenu("Netcode/" + nameof(NetworkAnimator))]
-    public class NetworkAnimator : NetworkBehaviour
+    public abstract class NetworkAnimator : NetworkBehaviour
```

Comment:
Please add automated tests for NetworkAnimator

Reply:
Sam and I had a conversation - he's fine with keeping the manual test for now, provided we add an automated soon.